code/controllers/subsystem/processing/quirks.dm

```diff
@@ -41,7 +41,3 @@ PROCESSING_SUBSYSTEM_DEF(quirks)
 			badquirk = TRUE
 	if(badquirk)
 		cli.prefs.save_character()
-
-	// Assign wayfinding pinpointer granting quirk if they're new
-	if(cli.calc_exp_type(EXP_TYPE_LIVING) < 1200 && !user.has_quirk(/datum/quirk/needswayfinder))
```

Comment:
cli.calc_exp_type(EXP_TYPE_CREW)

I led you astray here, EXP_TYPE_LIVING doesnt work with this proc. Crew however, will. I have confirmed it on the live servers.

Reply:
Actually replace the 1200 with the define 
20 HOURS 
as well. Magic numbers and all.